src/cmd-upload-oscontainer

```diff
@@ -89,8 +89,8 @@ if display_name == "":
     raise SystemExit(f"Failed to find NAME= in /usr/lib/os-release in commit {ostree_commit}")
 shutil.rmtree(tmp_osreleasedir)
 
-# The build ID is the container tag
-osc_name_and_tag = "{}:{}".format(args.name, latest_build)
+arch = meta.get("coreos-assembler.basearch", cmdlib.get_basearch)
+osc_name_and_tag = f"{args.name}:{latest_build}-{arch}"
```

Comment:
In order to allow safely "ratcheting" this change, maybe make it an option like `--arch-tag` and then the pipeline opts-in to it?

Reply:
Repushed